### PR TITLE
chore(format): fix formatting using prettier

### PR DESCRIPTION
**Pull request by anshumanv**
fix formatting using prettier

**What kind of change does this PR introduce?**
Formatted files using prettier.

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
NA

**Summary**
Fixed inconsistent file formatting

**Does this PR introduce a breaking change?**
No
**Other information**

### CODE REVIEW

**Comment by ematipico** [comment]
Actually, why don't we go further with this PR? We could add prettier to the eslint rules (eslint-plugin-prettier): https://github.com/prettier/eslint-plugin-prettier/blob/master/README.md#installation

**Comment by anshumanv** [comment]
That's a great suggestion @ematipico! On it. 👍

**Comment by anshumanv** [comment]
@evenstensberg @ematipico ptal ❄️

**Comment by anshumanv** [comment]
We're missing some definitions, would it make sense to also use `eslint-config-prettier`?

File: .eslintrc.js [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+		"prettier/prettier": "error",
 		"no-useless-escape": "off",
 		"quote-props": ["error", "as-needed"],
 		"no-dupe-keys": "error",
 		quotes: ["error", "double"],
 		"no-undef": "error",
 		"no-extra-semi": "error",
 		semi: "error",
-		"no-template-curly-in-string": "error",
+		"no-tempate-curly-in-string": "error",
```
Comment (ematipico) [inline .eslintrc.js]: `template`?

**Comment by ematipico** [comment]
@anshumanv yes I think so. Let's give it a spin!

**Comment by anshumanv** [comment]
On it, thanks for fixing the typo! :)

File: .eslintrc.js (hunk 1 above)
Comment (anshumanv) [reply .eslintrc.js]: My bad, thanks for the quick fix @evenstensberg ✨

**Comment by evenstensberg** [comment]
remember to pull, leaving it to you now

**Comment by anshumanv** [comment]
Configs seem to conflicts with prettier, had to disable them. Please suggest if better workarounds.

**Comment by evenstensberg** [comment]
Can you point at where you disabled them?

File: bin/config/config-yargs.js [hunk 2]
```diff
@@ -349,7 +349,8 @@ module.exports = function(yargs) {
 			},
 			p: {
 				type: "boolean",
-				describe: "shortcut for --optimize-minimize --define process.env.NODE_ENV=\"production\"",
+				// eslint-disable-next-line quotes
```
Comment (anshumanv) [inline bin/config/config-yargs.js]: Disabled

File: bin/utils/convert-argv.js [hunk 3]
```diff
@@ -24,7 +24,8 @@ module.exports = function(...args) {
 	}
 	if (argv.p) {
 		argv["optimize-minimize"] = true;
-		argv["define"] = [].concat(argv["define"] || []).concat("process.env.NODE_ENV=\"production\"");
+		// eslint-disable-next-line quotes
```
Comment (anshumanv) [inline bin/utils/convert-argv.js]: here

File: test/binCases/configFile/plugins-precedence/plugins-precedence.test.js [hunk 4]
```diff
@@ -15,7 +15,8 @@ test("plugins-precedence", () => {
 		"--target",
 		"async-node",
 		"--define",
-		"TEST=\"ok\""
+		// eslint-disable-next-line quotes
```
Comment (anshumanv) [inline test/binCases/configFile/plugins-precedence/plugins-precedence.test.js]: here

File: test/binCases/errors/info-verbosity/info-verbosity.test.js [hunk 5]
```diff
@@ -17,6 +17,7 @@ test("info-verbosity", () => {
 	]);
 
 	expect(stderr).toContain("Invalid values:");
-	expect(stderr).toContain("Argument: info-verbosity, Given: \"false\", Choices: \"none\", \"info\", \"verbose\"");
+	// eslint-disable-next-line quotes
```
Comment (anshumanv) [inline test/binCases/errors/info-verbosity/info-verbosity.test.js]: here

File: .eslintrc.js [hunk 6]
```diff
@@ -1,5 +1,6 @@
 module.exports = {
-	extends: ["eslint:recommended"],
+	extends: ["eslint:recommended", "plugin:prettier/recommended"],
```
Comment (anshumanv) [inline .eslintrc.js]: No need to add `prettier/prettier` in rules as the plugin does that for us

Comment (evenstensberg) [reply .eslintrc.js]: okay I think this is good

File: test/binCases/errors/info-verbosity/info-verbosity.test.js (hunk 5 above)
Comment (evenstensberg) [reply test/binCases/errors/info-verbosity/info-verbosity.test.js]: they're not conflicting, remove the lint disable, and use single quotes to escape, I think that might be a solution

File: test/binCases/configFile/plugins-precedence/plugins-precedence.test.js (hunk 4 above)
Comment (evenstensberg) [reply test/binCases/configFile/plugins-precedence/plugins-precedence.test.js]: SIngle char escape

File: bin/utils/convert-argv.js (hunk 3 above)
Comment (evenstensberg) [reply bin/utils/convert-argv.js]: Single char escape

File: bin/config/config-yargs.js (hunk 2 above)
Comment (evenstensberg) [reply bin/config/config-yargs.js]: Single char escape

File: test/binCases/errors/info-verbosity/info-verbosity.test.js (hunk 5 above)
Comment (anshumanv) [reply test/binCases/errors/info-verbosity/info-verbosity.test.js]: tried, not working, prettier fails after that :(

File: bin/config/config-yargs.js (hunk 2 above)
Comment (anshumanv) [reply bin/config/config-yargs.js]: prettier fails after that

File: bin/utils/convert-argv.js (hunk 3 above)
Comment (anshumanv) [reply bin/utils/convert-argv.js]: prettier fails after that

File: test/binCases/configFile/plugins-precedence/plugins-precedence.test.js (hunk 4 above)
Comment (anshumanv) [reply test/binCases/configFile/plugins-precedence/plugins-precedence.test.js]: prettier fails after that

File: test/binCases/errors/info-verbosity/info-verbosity.test.js (hunk 5 above)
Comment (evenstensberg) [reply test/binCases/errors/info-verbosity/info-verbosity.test.js]: Lemme see the ci failing on switching a single one

Comment (anshumanv) [reply test/binCases/errors/info-verbosity/info-verbosity.test.js]: Sure, thanks! 🌞

File: bin/cli.js [hunk 7]
```diff
@@ -107,9 +107,7 @@ For more information, see https://webpack.js.org/api/cli/.`);
 		 * used in place of process.stout
 		 */
 		const stdout = argv.silent
-			? {
-				write: () => {}
-			  } // eslint-disable-line
+			? { write: () => {} } // eslint-disable-line
```
Comment (ematipico) [inline bin/cli.js]: Do we have to keep the disable line here?

Comment (anshumanv) [reply bin/cli.js]: Nope, and good catch! :)

**Comment by anshumanv** [comment]
Leaving this to @evenstensberg for the final look ❄️ ✨

**Pull request merged**